core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -433,6 +434,26 @@ void validateDeleteManifest(ManifestFile manifest,
     Assert.assertFalse("Should find all files in the manifest", expectedFiles.hasNext());
   }
 
+  protected DataFile newDataFile(String partitionPath) {
+    return DataFiles.builder(table.spec())
+        .withPath("/path/to/data-" + UUID.randomUUID() + ".parquet")
+        .withFileSizeInBytes(10)
+        .withPartitionPath(partitionPath)
+        .withRecordCount(1)
+        .build();
+  }
+
+  protected DeleteFile newDeleteFile(int specId, String partitionPath) {
+    PartitionSpec spec = table.specs().get(specId);
+    return FileMetadata.deleteFileBuilder(spec)
+        .ofPositionDeletes()
+        .withPath("/path/to/delete" + UUID.randomUUID() + ".parquet")
```

Comment:
Nit: should be `delete-` to match `data-`.

Reply:
Fixed.